.env.testing

```diff
@@ -1,5 +1,5 @@
 APP_NAME="Wandersnap API"
-APP_ENV=testing
+APP_ENV=production
```

Comment:
production in testing? do we wanna keep this in?  :)

Reply:
done